server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/routes/WebSocketRoutes.scala

```diff
@@ -113,6 +120,10 @@ class WebSocketRoutes @Inject() (@Named(InjectionKeys.RFC_8621) val authenticato
         frame.content().readBytes(bytes)
         new String(bytes, StandardCharsets.UTF_8)
       })
+      .doOnNext(_ => {
+        requestCountMetric.increment()
+        requestThroughputGauge.setValue(requestCountMetric.meanRate().intValue)
```

Comment:
Kill that gauge, it won't be performant. And we can do a derivative for that in grafana.